spec/models/review_spec.rb

```diff
@@ -10,6 +10,13 @@
         review = Review.new(card_id: card.id, answer: "way")
         expect(review.check_translation).to be_truthy
       end
+      
+      it "increase review_count by one" do
+        review = Review.new(card_id: card.id, answer: "way")
+        expect {
+          review.check_translation
+        }.to change{card.reload.review_count}.by(1)
```

Comment:
Space missing to the left of {.<br>Space missing inside {.<br>Space missing inside }.